core/tests/performance_tests/splash_test.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the splash page"""
+
+import unittest
+from test_utils import TestBase
+
+
+class SplashPageTest(TestBase):
+    """Tests for `splash page`."""
+
+    # how do I call setUp only once before all the test cases.
```

Comment:
Hm, if a page is loading within required amount of time but, its total size is not below the threshold, then the tests should show it explicitly so that the developer can act accordingly. So, I think we should use `setUpClass` and a new pattern! WDYT?